driver/src/test/java/org/neo4j/driver/v1/integration/ServerKilledIT.java

```diff
@@ -43,19 +48,33 @@
  * Mainly concerned about the connection pool - we want to make sure that bad connections are evacuated from the
  * pool properly if the server dies, or all connections are lost for some other reason.
  */
+@RunWith(Parameterized.class)
 public class ServerKilledIT
 {
     @Rule
     public TestNeo4j neo4j = new TestNeo4j();
 
+    @Parameters(name = "{0} connections")
+    public static Collection<Object[]> data() {
```

Comment:
Can we make this method return a collection of `Config.EncryptionLevel`? I think it'll make code simpler - no need for ignored `testName` constructor argument and flat list of `Config.EncryptionLevel` can be returned.